### PR TITLE
Updated CI test to use pylint 2.2.2

### PR DESCRIPTION
**Pull request by joachimmetz**

### CODE REVIEW

**Comment by berggren** [comment]
@joachimmetz tests are failing with:
find: ‘tests’: No such file or directory

I don't use a single test directory in TS. Before this, nosetest discovered all tests. How are we doing test file discovery in this setup?

**Comment by berggren** [comment]
@joachimmetz #831 is merged now.

File: config/travis/install.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Script to set up Travis-CI test VM.
+#
+# This file is generated by l2tdevtools update-dependencies.py any dependency
+# related changes should be made in dependencies.ini.
+
+# Exit on error.
+set -e;
+if test ${TRAVIS_OS_NAME} = "linux" && test ${TARGET} != "jenkins";
+then
+	sudo rm -f /etc/apt/sources.list.d/travis_ci_zeromq3-source.list;
```
[the remark this answers is not in the document]
Comment (joachimmetz) [reply config/travis/install.sh]: yes this is to work around some older issue. I'll remove it and make a not to remove it entirely if no longer needed.

[the remark this answers is not in the document]
Comment (joachimmetz) [reply config/travis/install.sh]: https://github.com/log2timeline/l2tdevtools/issues/638

**Comment by joachimmetz** [comment]
> @joachimmetz #831 is merged now.

thx, I've changed pylint tests, path are hard coded in the runtests.sh script
removed the work-around and small changes to .pylintrc.

Let's see if the tests pass, I assume a couple of more iterations due to linter changes

**Comment by joachimmetz** [comment]
@berggren changes made with additional changes to get linter passing

**Pull request merged**